alvr/dashboard/src/dashboard/components/setup_wizard.rs

```diff
@@ -110,15 +122,66 @@ impl SetupWizard {
 Make sure you have at least one output audio device.",
                 |_| (),
             ),
-            Page::SoftwareRequirements => page_content(
+            Page::SoftwareRequirements => {
+                page_content(
+                    ui,
+                    "Software requirements",
+                    r"To stream the Quest microphone on Windows you need to install VB-Cable or Voicemeeter.
+On Linux, game audio and microphone might require pipewire and On connect/On disconnect script.
+Script is not 100% stable and might cause some instability issues with pipewire, but it should work.",
+                    |ui| {
+                        if ui.button("Download VB-Cable").clicked() {
+                            ui.ctx()
+                                .open_url(OpenUrl::same_tab("https://vb-audio.com/Cable/"));
+                        }
+                        if ui
+                            .button("'On connect/On disconnect' audio script")
+                            .clicked()
+                        {
+                            #[cfg(target_os = "linux")]
```

Comment:
Move the cfg gate before the if, so the button is hidden.

Reply:
added way more platform checks, but i have question a bit bellow about macos